client/web/src/enterprise/batches/detail/changesets/ExternalChangesetInfoCell.tsx

```diff
@@ -24,67 +23,55 @@ export const ExternalChangesetInfoCell: React.FunctionComponent<ExternalChangese
     node,
     viewerCanAdminister,
     className,
-}) => (
-    <div className={classNames('d-flex flex-column', className)}>
-        <div className="m-0">
-            <h3 className="m-0 d-block d-md-inline">
-                <LinkOrSpan
-                    to={node.externalURL?.url ?? undefined}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                    className="mr-2"
-                >
-                    {(isImporting(node) || importingFailed(node)) && (
-                        <>
-                            Importing changeset
-                            {node.externalID && <> #{node.externalID} </>}
-                        </>
-                    )}
-                    {!isImporting(node) && !importingFailed(node) && (
-                        <>
-                            {node.title}
-                            {node.externalID && <> (#{node.externalID}) </>}
-                        </>
-                    )}
-                    {node.externalURL?.url && (
-                        <>
-                            {' '}
-                            <ExternalLinkIcon size="1rem" />
-                        </>
+}) => {
+    const changesetTitle =
+        isImporting(node) || importingFailed(node) ? (
```

Comment:
Does eslint complain if we parenthesise the condition here? I've worked in too many languages with different precedence rules for ternaries to trust this. :grimacing: 

```suggestion
        (isImporting(node) || importingFailed(node)) ? (
```

Reply:
I think prettier NOPEs it if I try to add the parens. 😢

Reply:
No problem. Thanks for checking!